openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/SWIGTrainBoosterTest.java

```diff
@@ -28,15 +27,15 @@
  * Tests for the SWIGTrainResources class to ensure all resources are properly/safely initialized and released.
  *
  * @author Alberto Ferreira (alberto.ferreira@feedzai.com)
- * @since 1.0.18
+ * @since @@@since.next.release@@@ (renamed from SWIGTrainResourcesTest)
```

Comment:
I don't think this system works here. We'll need to hardcode the version

Reply:
So I should put the next version already? Hope we close this fast before a new one is released :sweat_smile:

Reply:
As soon as this is closed, it will go to master, and therefore will be available for release. I think that the problematic window is very small. In any way, you can sync with Hugo Abreu, as he opened a PR in the repo today.